includes/class-piklist-form.php

```diff
@@ -2149,13 +2149,13 @@ public static function validate_field_parameter($parameter, $value)
     {
       case 'capability':
 
-        return piklist_user::current_user_can($value);
+        return null === $value ? true : piklist_user::current_user_can($value);
```

Comment:
Rather than assert a literal null, let's just allow a falsey value to cover everything, so basically everything that _isn't_ a value is acceptable. Like so:
```
return empty($value) || piklist_user::current_user_role($value);
```